docker/base

```diff
@@ -14,6 +14,10 @@ echo 'APT::Install-Recommends "0";' > /etc/apt/apt.conf.d/99no-recommends
 # Make sure our packages list is updated
 apt update
 
+# Get the current distribution name
+apt -y install lsb-release
+dist="$(lsb_release -cs)"
```

Comment:
This belongs in the previous commit (where you (un)fix `apt-preferences`).

Reply:
Wait, why this? You are already installing `lsb_relase` in the line 22.

Reply:
Mixed up commits!

Reply:
Updated.